nanoc.yaml

```diff
@@ -65,17 +65,25 @@ data_sources:
     type: repo_docs
     items_root: /docs/prometheus/latest/
     config:
-      name: 'latest (2.2)'
+      name: 'latest (2.3)'
       repository: https://github.com/prometheus/prometheus.git
-      refspec: release-2.2
+      refspec: release-2.3
+  -
+    type: repo_docs
+    items_root: /docs/prometheus/2.3/
+    config:
+      name: '2.3'
+      repository: https://github.com/prometheus/prometheus.git
+      refspec: release-2.3
+      canonical: /docs/prometheus/latest/
   -
     type: repo_docs
     items_root: /docs/prometheus/2.2/
     config:
       name: '2.2'
       repository: https://github.com/prometheus/prometheus.git
       refspec: release-2.2
-      canonical: /docs/prometheus/latest/
```

Comment:
@brancz I acknowledge that's confusing and redundant, but the canonical page for all docs pages should be latest. This attribute is used to set the meta canonical tag to prevent double indexing issues in google/search engines.

Reply:
I see, thanks for clarifying. I browsed the netlify test deploy and all was good so I assumed that it was correct. Thanks!